more/release-notes.mdx

```diff
@@ -13,6 +13,22 @@ Literal AI cloud is currently compatible with:
 * [TypeScript SDK](/typescript-client/development) version `0.0.503` and above.
 </Note>
 
+# v0.0.619
+
+## New features
```

Comment:
We need three pounds to have correct indentation in Mintlify. It's weird but no other way...
<img width="300" alt="image" src="https://github.com/user-attachments/assets/3d39db9e-6a2e-4594-b3d9-1a32275afe12">